content/dart/concepts/iterable/iterable.md

```diff
@@ -0,0 +1,66 @@
+---
+Title: 'Iterable'
+
+Description: '**Iterable** is an object which can be looped over or iterated over with the help of a for loop. Objects like lists, tuples, sets, dictionaries, strings, etc. are called iterables. In short and simpler terms, iterable is anything that you can loop over.'
+Subjects:
+  - Data Science
+Tags:
+  - 'Data'
+  - 'Objects'
+  - 'Loops'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-example-course'
+  - 'paths/example-path'
+---
+
+An **iterable** is a collection of elements that can be accessed sequentially. Dart uses the Iterable<E> abstract class to represent iterable objects.
+
+An iterable object has an iterator getter that returns an iterator object. The iterable object uses the iterator to step through its elements.
+
+Each time you access the iterator getter, the iterable object returns a new iterator. For this reason, you can create more than one iterator from the same iterable object.
+
+If an object is iterable, you can use the for-in statement to iterate over its elements.
+
+## Syntax
+
+[Text, code, images, parameters, etc. about the syntax]
+
+## Example
+
+Since List and Set are iterable, you can use the for-in statement to iterate over their elements. For example:
+
+void main() {
+Iterable<int> iterable = [1, 2, 3, 4, 5];
+for (var n in iterable) {
+print(n);
+}
+}
+Code language: Dart (dart)
```

Comment:
````suggestion
```dart
void main() {
  Iterable<int> iter = [12, 23, 34, 45, 56];
  for (var num in iter) {
    print(num);
  }
}
```
````